File: 프로그래머스/unrated/181878. 원하는 문자열 찾기/원하는 문자열 찾기.py
def solution(myString, pat):
    answer = 0
    str1 = myString.upper()
    pat1 = pat.upper()
    for i in range(len(str1)):
        if(str1[i : i + len(pat1)] == pat1):
            answer = 1
    
    return answer

File: 프로그래머스/0/181878. 원하는 문자열 찾기/원하는 문자열 찾기.js
function solution(myString, pat) {
    var answer = 0;
    var mSize = myString.length;
    var size = pat.length;
    

    
    for(var i =0; i < mSize - size+1; i++){
        var str = myString.substring(i, i + size);
        var uStr = str.toUpperCase();
        var pStr = pat.toUpperCase();
        // console.log(uStr + " " +  pStr)
        if(uStr === pStr){
            return 1;
        }
        
    }
    
    return answer;
}